packages/storybook-vue/src/stories/Article/argsTypes.ts

```diff
@@ -0,0 +1,8 @@
+export const componentArgs = {
+  default: `<Heading :level="1">Lorem ipsum</Heading>
+  <paragraph>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+  magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+  consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+  pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
+  laborum.</paragraph>`,
```

Comment:
There is a slight inconsistency: `Heading` is capitalized, while `paragraph` is lowercase.